docs/book/src/reference/jobs.md

```diff
@@ -19,12 +19,18 @@ Prow Presubmits:
 * [pull-cluster-api-verify-main] `./scripts/ci-verify.sh`
 * [pull-cluster-api-test-main] `./scripts/ci-test.sh`
 * [pull-cluster-api-test-mink8s-main] `./scripts/ci-test.sh`
+  * KUBEBUILDER_ENVTEST_KUBERNETES_VERSION: `1.24.2`
+* [pull-cluster-api-e2e-mink8s-main] `./scripts/ci-test.sh`
```

Comment:
```suggestion
* ✳️️ ✴️ [pull-cluster-api-e2e-mink8s-main] `./scripts/ci-test.sh`
```

Reply:
Hm. WDYT about changing this whole section to: (+ preserving the sub items)

* mandatory for merge, always run:
  * [pull-cluster-api-build-main]
  * [pull-cluster-api-verify-main]

* mandatory for merge, run if go code changes:
  * [pull-cluster-api-test-main]
  * [pull-cluster-api-e2e-main]

* optional for merge, run if go code changes
  * [pull-cluster-api-apidiff-main]
  * [pull-cluster-api-e2e-informing-main]

* optional for merge, run if manually triggered
  * [pull-cluster-api-test-mink8s-main]
  * [pull-cluster-api-e2e-mink8s-main]
  * [pull-cluster-api-e2e-ipv6-main]
  * [pull-cluster-api-e2e-full-main]
  * [pull-cluster-api-e2e-full-dualstack-ipv6-main]
  * [pull-cluster-api-e2e-workload-upgrade-1-27-latest-main]
  * [pull-cluster-api-e2e-scale-main-experimental]

I think that would be more intuitive

Reply:
Great idea, updated now1